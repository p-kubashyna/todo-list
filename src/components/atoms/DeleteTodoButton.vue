<script setup></script>

<template>
  <button class="delete-button">
    <img
      class="delete-button-image"
      src="../../assets/images/trashcan_icon.png"
      alt="delete icon"
    />
  </button>
</template>

<style scoped>
.delete-button {
  width: 25px;
  background-color: white;
  border-width: 0;
  margin-left: auto;
}

.delete-button-image {
  height: 17px;
  width: 17px;
}
</style>
