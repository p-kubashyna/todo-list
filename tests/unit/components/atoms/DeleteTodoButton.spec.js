import { describe, expect, it } from "@jest/globals";
import { shallowMount } from "@vue/test-utils";
import DeleteTodoButton from "@/components/atoms/DeleteTodoButton.vue";

describe("Given the DeleteTodoButton component", () => {
  it("should render", () => {
    const wrapper = shallowMount(DeleteTodoButton, {});

    expect(wrapper.element).toMatchSnapshot();
  });
});
