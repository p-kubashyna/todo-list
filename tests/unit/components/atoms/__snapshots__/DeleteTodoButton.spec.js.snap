// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the DeleteTodoButton component should render 1`] = `
<button
  class="delete-button"
>
  <img
    alt="delete icon"
    class="delete-button-image"
    src=""
  />
</button>
`;
